data_steward/utils/participant_summary_requests.py

```diff
@@ -157,6 +172,77 @@ def get_deactivated_participants(project_id, dataset_id, tablename, columns):
     return df
 
 
+def get_site_participant_information(project_id, hpo_id):
+    """
+    Fetches the necessary participant information for a particular site.
+
+    :param project_id: The RDR project that contains participant summary data
+    :param hpo_id: awardee name of the site
+
+    :return: returns dataframe of participant information
+    """
+    # Parameter checks
+    if not isinstance(project_id, str):
+        raise RuntimeError(f'Please specify the RDR project')
+
+    if not isinstance(hpo_id, str):
+        raise RuntimeError(f'Please provide an hpo_id')
+
+    token = get_access_token()
+
+    headers = {
+        'content-type': 'application/json',
+        'Authorization': f'Bearer {token}'
+    }
+
+    # Make request to get API version. This is the current RDR version for reference see
+    # see https://github.com/all-of-us/raw-data-repository/blob/master/opsdataAPI.md for documentation of this API.
+    # consentForElectronicHealthRecords=SUBMITTED -- ensures only consenting participants are returned via the API
+    #   regardless if there is EHR data uploaded for that participant
+    # suspensionStatus=NOT_SUSPENDED and withdrawalStatus=NOT_WITHDRAWN -- ensures only active participants returned
+    #   via the API
+    url = 'https://{0}.appspot.com/rdr/v1/ParticipantSummary?awardee={1}&suspensionStatus={2}&consentForElectronicHealthRecords={3}&withdrawalStatus={4}&_sort=participantId&_count=1000'.format(
+        project_id, hpo_id, 'NOT_SUSPENDED', 'SUBMITTED', 'NOT_WITHDRAWN')
```

Comment:
This line is too long.  Please consider reformatting and using f-strings.
```suggestion
    url = (f'https://{project_id}.appspot.com/rdr/v1/ParticipantSummary'
           f'?awardee={hpo_id}'
           f'&suspensionStatus=NOT_SUSPENDED'
           f'&consentForElectronicHealthRecords=SUBMITTED'
           f'&withdrawalStatus=NOT_WITHDRAWN'
           f'&_sort=participantId'
           f'&_count=1000'
          )
```